app/src/main/java/org/schabi/newpipe/player/MainVideoPlayer.java

```diff
@@ -392,6 +402,18 @@ public void onPlaybackParameterChanged(float playbackTempo, float playbackPitch,
         private RelativeLayout windowRootLayout;
         private View secondaryControls;
 
+        private int maxGestureLength;
+
+        {
+            DisplayMetrics m = getResources().getDisplayMetrics();
```

Comment:
What do you think of using the `rootView`'s width and height?

The code would be simpler, and using this approach, we could already support the multi window mode just by adding a listener to the `rootView` layout changes, that when we detect one, we could simply update the variable and the gesture would work correctly in the new size.

Reply:
Okay, will do.